server/controllers/donor.controller.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const { Donor, Event, Gift, User, Profile } = require('../../config/db');
+const { messages } = require('./../helper');
+
+module.exports = {
+  create(req, res) {
+    let assignDonor = Object.assign({}, req.body, {gift_id: req.params.gift_id},
+      {user_id: req.params.id});
+    Gift.findOne({
+      where: {
+        id: req.params.gift_id,
```

Comment:
at the beginning i have only two parameters id for user, and gift_id. Therefore, i can get event_id only from Event where id: gift.dataValues.event_id. i did it in a code below...How i can add event id in gift if i can only retrieve all about event from Gift???